docs/topics/settings.rst

```diff
@@ -286,9 +288,20 @@ DEPTH_PRIORITY
 
 Default: ``0``
 
-An integer that is used to adjust the request priority based on its depth.
+Scope: ``scrapy.spidermiddlewares.depth.DepthMiddleware``
+
+An integer that is used to adjust the request priority based on its depth:
+
+- **a positive value will decrease the priority**
```

Comment:
Can we be even more explicit? E.g. "it means requests with larger depth will be processed later".